aws/resource_aws_dlm_lifecycle_policy_test.go

```diff
@@ -296,6 +300,7 @@ resource "aws_dlm_lifecycle_policy" "full" {
       tags_to_add {
         tf-acc-test-added = "full-updated"
       }
+      copy_tags = true
```

Comment:
Can you check that `false` works as well? Either in the initial test configuration or the updated test configuration. Thanks!